foursight_core/app_utils.py

```diff
@@ -36,9 +36,6 @@ class AppUtilsCore:
     so that a class named AppUtils is easier to define in libraries that import foursight-core.
     """
 
-    # dmichaels/2022-07-20/C4-826: Apply identity globally.
-    apply_identity_globally()
```

Comment:
I think we roll with this assuming the change works, which I think in `app.py` it will. You might consider leaving it commented out as a holdover and explaining why it was moved.